testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksUnityCompatibility.java

```diff
@@ -0,0 +1,224 @@
+    public void testBasicTableReadWrite()
+    {
+        String schemaName = "test_delta_basic_" + randomNameSuffix();
+        String tableName = "test_dt_basic_" + randomNameSuffix();
+        String schemaLocation = getLocationForSchema(externalLocationPath, schemaName);
+        String tableLocation = getLocationForTable(externalLocationPath, schemaName, tableName);
+        try {
+            onDelta().executeQuery(
+                    format("CREATE SCHEMA %s.%s MANAGED LOCATION '%s'",
+                            unityCatalogCatalogName, schemaName, schemaLocation));
+            onDelta().executeQuery(
+                    format("CREATE TABLE %s.%s.%s (c1 int, c2 string) using delta location '%s'",
+                            unityCatalogCatalogName, schemaName, tableName, tableLocation));
+            onDelta().executeQuery(
+                    format("INSERT INTO %s.%s.%s values (1, 'one')",
+                            unityCatalogCatalogName, schemaName, tableName));
+
+            assertThat(onTrino().executeQuery("SHOW SCHEMAS FROM delta"))
+                    .contains(ImmutableList.of(
+                            row(schemaName.toLowerCase(ENGLISH))));
+
+            assertThat(onTrino().executeQuery("SHOW TABLES in delta." + schemaName))
+                    .containsOnly(row(tableName.toLowerCase(ENGLISH)));
+
+            // select
+            List<QueryAssert.Row> expectedRowsForSelect = ImmutableList.of(row(1, "one"));
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.%s.%s", schemaName, tableName)))
+                    .containsOnly(expectedRowsForSelect);
+
+            // insert
+            List<QueryAssert.Row> expectedRowsForInsert = ImmutableList.of(row(1, "one"), row(2, "two"));
+            onTrino().executeQuery(format("INSERT INTO delta.%s.%s values (2, 'two')", schemaName, tableName));
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.%s.%s order by c1", schemaName, tableName)))
+                    .containsOnly(expectedRowsForInsert);
+            assertThat(onDelta().executeQuery(format("SELECT * FROM %s.%s.%s order by c1", unityCatalogCatalogName, schemaName, tableName)))
+                    .containsOnly(expectedRowsForInsert);
+
+            // update
+            List<QueryAssert.Row> expectedRowsForUpdate = ImmutableList.of(row(1, "one"), row(2, "two hundred"));
+            onTrino().executeQuery(format("UPDATE delta.%s.%s SET c2 = 'two hundred' WHERE c1 = 2", schemaName, tableName));
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.%s.%s order by c1", schemaName, tableName)))
+                    .containsOnly(expectedRowsForUpdate);
+            assertThat(onDelta().executeQuery(format("SELECT * FROM %s.%s.%s order by c1", unityCatalogCatalogName, schemaName, tableName)))
+                    .containsOnly(expectedRowsForUpdate);
```

Comment:
`containsOnly` doesn't verify the order. Please remove `order by` from SELECT statement.